src/main/scala/mesosphere/marathon/core/launcher/impl/TaskOpFactoryImpl.scala

```diff
@@ -176,7 +179,11 @@ class TaskOpFactoryImpl(
         agentId = Some(offer.getSlaveId.getValue),
         attributes = offer.getAttributesList.asScala
       ),
-      reservation = Task.Reservation(persistentVolumeIds, Task.Reservation.State.New(timeout = Some(timeout)))
+      reservation = Task.Reservation(persistentVolumeIds, Task.Reservation.State.New(timeout = Some(timeout))),
+      status = Task.Status(
+        stagedAt = now,
```

Comment:
it becomes very obvious now, that a mandatory `stagedAt` is a lie 😁 
But that can be fixed in the next step as far as in concerns me